apps/omg/test/omg/dependency_conformance/signature_test.exs

```diff
@@ -18,93 +18,117 @@ defmodule OMG.DependencyConformance.SignatureTest do
   by both Elixir signature code and contract signature code.
   """
 
-  alias OMG.DevCrypto
   alias OMG.Eth
   alias OMG.State.Transaction
   alias OMG.TestHelper
-  alias Support.Deployer
-  alias Support.DevNode
 
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
   use ExUnit.Case, async: false
 
   @moduletag :integration
   @moduletag :common
 
-  @alice TestHelper.generate_entity()
-  @bob TestHelper.generate_entity()
+  @alice %{
+    addr: <<215, 32, 17, 47, 111, 72, 20, 47, 149, 226, 138, 242, 35, 254, 141, 212, 16, 22, 155, 182>>,
+    priv:
+      <<170, 145, 170, 111, 112, 29, 60, 152, 73, 136, 133, 220, 101, 57, 32, 144, 174, 192, 102, 193, 186, 145, 231,
+        104, 132, 231, 27, 63, 128, 36, 204, 94>>
+  }
+  @bob %{
+    addr: <<141, 246, 138, 77, 76, 3, 78, 54, 173, 40, 234, 195, 29, 170, 154, 64, 99, 14, 118, 139>>,
+    priv:
+      <<6, 31, 86, 177, 209, 153, 18, 204, 55, 88, 137, 149, 48, 164, 92, 147, 255, 58, 163, 80, 243, 202, 105, 56, 176,
+        216, 149, 207, 188, 96, 160, 87>>
+  }
   @eth OMG.Eth.RootChain.eth_pseudo_address()
-  @token TestHelper.generate_entity().addr
+  @token <<235, 169, 32, 193, 242, 237, 159, 137, 184, 46, 124, 13, 178, 171, 61, 87, 179, 179, 135, 146>>
 
-  setup_all do
-    {:ok, exit_fn} = DevNode.start()
-
-    root_path = Application.fetch_env!(:omg_eth, :umbrella_root_dir)
-    {:ok, [addr | _]} = Ethereumex.HttpClient.eth_accounts()
+  setup %{} do
+    vcr_path = Path.join(__DIR__, "../../fixtures/vcr_cassettes")
+    ExVCR.Config.cassette_library_dir(vcr_path)
 
-    {:ok, _, signtest_addr} = Deployer.create_new("PaymentEip712LibMock", root_path, Eth.Encoding.from_hex(addr), [])
+    :ok
+  end
 
-    # impose our testing signature contract wrapper (mock) as the validating contract, which normally would be
-    # plasma framework
+  setup_all do
+    %{plasma_framework: plasma_framework} = Application.get_env(:omg_eth, :contract_addr)
```

Comment:
There could be more key-values in this map. The way you restore it after test, only `plasma_framework` gets restored.
```suggestion
    contract_addresses = Application.get_env(:omg_eth, :contract_addr)
```
and restore :point_down: